matlab_simulation/05-estimation/runFastSLAM.m

```diff
@@ -0,0 +1,61 @@
+%% Fast SLAM Example
+% Author: Bismaya Sahoo.
+%% Initializations
+clear all; close all; clc;
+% Primary Initializations           
+poseInitial=[0,0,0]';%initial Pose
+dt=0.1; timeInterval=dt;%time interval
+finalTime=20;%final Time in sec
+MotionNoise=[0.001 0 0;0 0.001 0;0 0 0.0001];R=MotionNoise;
+MeasurementNoise=[0.02 0;0 0.02];Q=MeasurementNoise;
+noFeatures=20;% No of features in Map
+ParticleSize=100;%No of particles
+rmax = 10;
+thmax = pi/4;
+u=[1;0.3];
+%Fast Slam Specific Initializations
```

Comment:
Separate lines with more "%" might be more visually clear and easy to understand.

`% Initial the robot pose`
`pose(:,1) = poseInitial;`

`% xxxxx`
 `totalParticles=ParticleSize;`
 `particleSet=zeros(size(poseInitial,1),totalParticles);`

`% Mean of xxx`
 `muFeat=zeros(size(Q,1),noFeatures,totalParticles);`
 `muFeatPred=muFeat;`

`% Covariance of xxx`
 `covFeat=zeros(size(Q,1),size(Q,1),noFeatures,totalParticles);`
 `covFeatPred=covFeat;`